src/gbd_mapping_generator/coverage_gap_builder.py

```diff
@@ -67,7 +67,7 @@ def make_coverage_gap(name, gbd_id, distribution, restrictions, levels=None, aff
         elif field:
             out += 2*TAB + f"{field_name}={field},\n"
 
-    for entity_type, entity_list in zip(['causes', 'risks'], [affected_causes, affected_risks]):
+    for entity_type, entity_list in zip(['causes', 'risk_factors'], [affected_causes, affected_risks]):
```

Comment:
If we're going to switch, switch everywhere.  Do a find and replace for affected_risks to affected_risk_factors